src/python/pcmdi/meanabs_xyt.py

```diff
@@ -0,0 +1,17 @@
+import MV2
+import cdutil
+
+def compute(dm, do):
+    """ Computes Mean Absolute Error"""
+    if dm is None and do is None:  # just want the doc
+        return {
+            "Name": "Mean Absolute Error",
+            "Abstract": "Compute Full Average of " +
+            "Absolute Difference Between Model And Observation",
+            "Contact": "pcmdi-metrics@llnl.gov",
+        }
+    absdif = MV2.absolute(MV2.subtract(dm, do))
+    mae = cdutil.averager(absdif,axis='xyt',weights='weighted')
```

Comment:
@gleckler1 there is a meanabsdiff in genutil.statistics and it has an option `centered=1` do we care about this here or not?